### PR TITLE
Update ncar-ncl.rb

### PR DESCRIPTION
**Pull request by core-code**
== is not something that is normally used in caskfiles

### CODE REVIEW

File: Casks/ncar-ncl.rb [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
   end
 
   appcast "https://www.ncl.ucar.edu/current_release.shtml"
   name "NCAR Command Language"
   name "ncl"
+  desc "Interpreted language for scientific data analysis and visualization"
   homepage "https://www.ncl.ucar.edu/"
 
-  depends_on x11: true
+  depends_on cask: "xquartz"
```
Comment (vitorgalvao) [inline Casks/ncar-ncl.rb:18]: @miccal Why the change? Was it failing before? These are not equivalent; `depends_on x11` is meant to work even if the user hasn’t installed through a cask.

Comment (miccal) [reply Casks/ncar-ncl.rb:18]: > Was it failing before?

Yes:
```
Error: Cask 'ncar-ncl' requires XQuartz/X11, which can be installed using Homebrew Cask by running:
  brew cask install xquartz
or manually, by downloading the package from:
  https://www.xquartz.org/
Error: Process completed with exit code 1.
```
which come from [here](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/cask/exceptions.rb#L107-L120).

I made the same change in https://github.com/Homebrew/homebrew-cask/pull/90345

I can revert these changes, but then a `ci-skip-install` label may be required to pass CI.

Comment (miccal) [reply Casks/ncar-ncl.rb:18]: Thought about it, and I will revert this.

**Review by miccal — APPROVED** [review]

**Pull request merged**